### PR TITLE
Note about the text cursor, and about pressing the enter key

### PR DESCRIPTION
**Pull request by chuseq**
Added a short note about the text cursor after the prompt. Sometimes we have found that some new learners don't infer what that rectangle is (maybe worth mentioning in the Background paragraph that in a GUI the graphical cursor is the mouse pointer? i don't know). Also added that they have to press the enter key to execute a command. 
Maybe it's obvious but for people really new to a shell can be helpful.

### CODE REVIEW

File: _episodes/01-intro.md [hunk 1]
```diff
@@ -72,6 +72,11 @@ In the examples for this lesson, we'll show the prompt as `$ `.
 Most importantly:
 when typing commands, either from these lessons or from other sources,
 *do not type the prompt*, only the commands that follow it.
+Also note that after you type a command, you have to press the *Enter* key yo execute it.
```
Comment (gdevenyi) [inline _episodes/01-intro.md:75]: Can you please convert the Enter code to use the `<kbd>` syntax.

File: _episodes/01-intro.md [hunk 2]
```diff
@@ -72,6 +72,11 @@ In the examples for this lesson, we'll show the prompt as `$ `.
...
+
+The prompt is followed by a **text cursor**, a character that indicates the position where your typing will appear. 
+The cursor is usually a flashing solid rectangle, but it can also be an underscore or a pipe.
```
Comment (gdevenyi) [inline _episodes/01-intro.md:78]: Perhaps here a "flashing or solid block".

I wonder if we can actually put in the actual text characters instead?

**Review by gdevenyi — CHANGES_REQUESTED** [review]

**Comment by gdevenyi** [comment]
Thanks for the suggestions, I like this addition, but I have a few changes to request.

File: _episodes/01-intro.md (hunk 1 above)
Comment (chuseq) [reply _episodes/01-intro.md]: Done. Also corrected s/yo/to/.

File: _episodes/01-intro.md (hunk 2 above)
Comment (chuseq) [reply _episodes/01-intro.md]: Done. I was looking for an ascii code or something for the block cursor but could not find any.

**Comment by gdevenyi** [comment]
Thanks!

**Pull request merged**